native/chat/chat-thread-background.react.js

```diff
@@ -15,6 +20,35 @@ type BackgroundChatThreadListProps = {
 export default function BackgroundChatThreadList(
   props: BackgroundChatThreadListProps,
 ) {
+  const chatThreadListData = useSelector(state => chatListData(state));
+
+  const findUnreadBackgroundThreadsNumber = (
```

Comment:
Why define this function separately, instead of just inline in the `useMemo` below?

(Based on my comment below we'll want to keep it separate so we can use it in `web`. But in this initial implementation where we were defining it inline in `BackgroundChatThreadList`, I think it would've made sense to just put it inside the `useMemo`.)